api/src/main/java/com/example/api/model/activity/result/SurveyResult.java

```diff
@@ -33,7 +34,7 @@ public class SurveyResult extends TaskResult{
 
     @Override
     public boolean isEvaluated() {
-        return this.getPointsReceived() != null;
+        return !Objects.equals(this.getPointsReceived(), survey.getMaxPoints());
```

Comment:
Jezeli uzyskane punkty z ankiety nie rownaja sie maxPoints to wtedy jest evaluated? nie rozumiem

Reply:
Ogólnie jest tak, że z ankiety możesz dostać tylko maksymalną liczbę punktów (dostajesz je po wypełnieniu ankiety). Przemyślałem to jednak i może lepiej sprawdzać to czy data wysłania nie jest nullem i liczba punktów nie jest nullem (no bo może być sytuacja, że student dostanie maksa za ankiete ale po tym liczba maks punktów w ankiecie się zmieni i wtedy jego SurveyResult nie będzie nigdzie brany pod uwagę bo isEvaluated()  = false